modules/cnv_viewing_guest_ip_addresses.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv_users_guide.adoc
+
+[id='viewing-guest-ip-addresses-{context}']
+=== Viewing guest IP addresses
+
+You can view the IP addresses that are assigned to your Linux virtual machines
+by installing a QEMU guest agent on the host. When the guest agent is running,
+you can view the virtual machine IP addresses for each interface by checking the
+VMI status on the command line.
+
+.Procedure
+
+. Install the QEMU guest agent on the host:
++
+----
+$ yum install -y qemu-guest-agent
+----
+
+. Start the QEMU guest agent service:
++
+----
+$ systemctl start qemu-guest-agent
+----
+
+. View the IP address information for a VMI:
++
+----
+$ oc describe vmi <vmi_name>
+
+...
+Interfaces:
+   Interface Name:  eth0
+   Ip Address:      10.244.0.37/24
+   Ip Addresses:
+     10.244.0.37/24
+     fe80::858:aff:fef4:25/64
+   Mac:             0a:58:0a:f4:00:25
+   Name:            default
+   Interface Name:  v2
+   Ip Address:      1.1.1.7/24
+   Ip Addresses:
+     1.1.1.7/24
+     fe80::f4d9:70ff:fe13:9089/64
+   Mac:             f6:d9:70:13:90:89
+   Interface Name:  v1
+   Ip Address:      1.1.1.1/24
+   Ip Addresses:
+     1.1.1.1/24
+     1.1.1.2/24
+     1.1.1.4/24
+     2001:de7:0:f101::1/64
+     2001:db8:0:f101::1/64
+     fe80::1420:84ff:fe10:17aa/64
+   Mac:             16:20:84:10:17:aa
+----
++
+[NOTE]
+====
+You can also view the IP address information by running `ip addr` on the host,
```

Comment:
The host here is the VM? You mean connect to the VM and run ip addr?

Reply:
@myakove  I actually thought the host was *not* the VM, based on the output that @mmirecki provided in https://jira.coreos.com/browse/CNV-1019 but I could be wrong. Marcin, can you clarify this?

Reply:
The "host" is the VM.
I think Meni is right here that "host" might be ambiguous.
It will be better to change:
"installing a QEMU guest agent on the host."   -->   "installing a QEMU guest agent on the VM."
" Install the QEMU guest agent on the host:"    -->   " Install the QEMU guest agent on the VM:"
"by running `ip addr` on the host,"         -->  "by running `ip addr` on the VM"

Reply:
Thanks @mmirecki! That was really helpful. I made the changes in my recent commit: https://github.com/openshift/openshift-docs/pull/14075/commits/3a6d0cdbb10cc01a01c4458e29e2c8f00e14dd39

@myakove I don't see a QE sub-task on this story. How will I find out when this content passes QE? :) Good catch on the ambiguous references to "host," btw.

Reply:
@ousleyp We don't have this under our sprint so I didn't create any sub-task.